scan/lib/scan/options.rb

```diff
@@ -70,19 +70,30 @@ def self.available_options
                                      type: Boolean,
                                      optional: true),
 
-        # reset simulator
```

Comment:
this structuring comment is not valid any more as `force_quit_simulator` was added and "belongs" to `reset_simulator` thematically, correct? If so, what would be a new description of what these to do? "before simulator usage" maybe?